origin_ui/src/app/(dashboard)/config/page.tsx

```diff
@@ -0,0 +1,379 @@
+        TopologyNamespaceUrl: string;
+        DirectorUrl: string;
+        NamespaceUrl: string;
+        JwkUrl: string;
+    }
+    Client: {
+        StoppedTransferTimeout: number;
+        SlowTransferRampupTime: number;
+        SlowTransferWindow: number;
+        DisableHttpProxy: boolean;
+        DisableProxyFallback: boolean;
+        MinimumDownloadSpeed: number;
+    }
+    DisableHttpProxy: boolean
+    DisableProxyFallback: boolean;
+    MinimumDownloadSpeed: number;
+    Origin: {
+        Url: string;
+        ExportVolume: string;
+        NamespacePrefix: string;
+        Multiuser: boolean;
+        UseCmsd: boolean;
+        UIPasswordFile: string;
+        SelfTest: boolean;
+    };
+    Director: {
+        DefaultResponse: string;
+        MaxMindKeyFile: string;
+        GeoIPLocation: string;
+    };
+    Registry: {
+        DbLocation: string;
+    };
+    Server: {
+        TLSCertificate: string;
+        TLSCACertificateFile: string;
+        TLSCACertificateDirectory: string;
+        TLSCAKey: string;
+        TLSKey: string;
+        Port: number;
+        Address: string;
+        ExternalAddress: string;
+        Hostname: string;
+        IssuerJwks: string;
+    };
+    OIDC: {
+        ClientIDFile: string;
+        ClientSecretFile: string;
+        DeviceAuthEndpoint: string;
+        TokenEndpoint: string;
+        UserInfoEndpoint: string;
+    };
+    Xrootd: {
+        Port: number;
+        RunLocation: string;
+        RobotsTxtFile: string;
+        ScitokensConfig: string;
+        Mount: string;
+        MacaroonsKeyFile: string;
+        Authfile: string;
+        ManagerHost: string;
+        SummaryMonitoringHost: string;
+        DetailedMonitoringHost: string;
+        LocalMonitoringHost: string;
+        Sitename: string;
+    };
+    Monitoring: {
+        DataLocation: string;
+        PortLower: number;
+        PortHigher: number;
+    };
+}
+
+function sortConfig (a: any, b: any) {
+    if(typeof a[1] == 'object' && typeof b[1] != 'object'){
+        return 1
+    }
+    if(typeof a[1] != 'object' && typeof b[1] == 'object'){
+        return -1
+    }
+    return a[0].localeCompare(b[0])
+}
+
+
+interface ConfigDisplayProps {
+    id: string
+    name: string
+    value: Partial<Config> | string | number | boolean | any
+    level: number
+}
+
+function ConfigDisplay({id, name, value, level = 1}: ConfigDisplayProps) {
+
+    id = `${id}-${name}`
+    let formElement = undefined
+
+    if(typeof value === 'string' || value instanceof String){
+
+        // For visual consistency convert empty strings to a blank space
+        value = value === "" ? " " : value
+
+        formElement = <TextField
+            fullWidth
+            disabled
+            size="small"
+            id={`${id}-text-input`}
+            label={name}
+            variant={"outlined"}
+            value={value}
+        />
+    }
+
+    if(typeof value === 'number' || value instanceof Number){
```

Comment:
Is it possible that we add unit for the values under `Transport`? Right now having 10 digit number as values for various "duration" fields can confuse people, or we might want to convert the durations into second when displaying them (I can do the conversion at the backend if that works better)

<img width="726" alt="Screenshot 2023-11-02 at 9 07 57 AM" src="https://github.com/PelicanPlatform/pelican/assets/41393704/8e957981-7fa5-43ae-9b9a-dbe1fc469b65">